src/Commands/RestoreCommand.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+/*
+ * This file is part of the hyn/multi-tenant package.
+ *
+ * (c) Daniël Klabbers <daniel@klabbers.email>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ *
+ * @see https://laravel-tenancy.com
+ * @see https://github.com/hyn/multi-tenant
+ */
+
+namespace Hyn\Tenancy\Commands;
+
+use Hyn\Tenancy\Models\Website;
+use Illuminate\Console\Command;
+use Hyn\Tenancy\Traits\DispatchesEvents;
+use Hyn\Tenancy\Events\Websites as Events;
+
+class RestoreCommand extends Command
+{
+    use DispatchesEvents;
+
+    /**
+     * The name and signature of the console command.
+     *
+     * @var string
+     */
+    protected $signature = 'tenancy:restore';
```

Comment:
Not quite sure about the naming.. it's not restoring, but recreating the databases. If I see "restore", I am thinking of restoring a backup or something, not just recreating the database.